ports/rp2/boards/SPARKFUN_XRP_CONTROLLER_BETA/pins.csv

```diff
@@ -0,0 +1,93 @@
+# All GPIO pins available to user
+GP0,GPIO0
+GP1,GPIO1
+GP2,GPIO2
+GP3,GPIO3
+GP4,GPIO4
+GP5,GPIO5
+GP6,GPIO6
+GP7,GPIO7
+GP8,GPIO8
+GP9,GPIO9
+GP10,GPIO10
+GP11,GPIO11
+GP12,GPIO12
+GP13,GPIO13
+GP14,GPIO14
+GP15,GPIO15
+GP16,GPIO16
+GP17,GPIO17
+GP18,GPIO18
+GP19,GPIO19
+GP20,GPIO20
+GP21,GPIO21
+GP22,GPIO22
+GP26,GPIO26
+GP27,GPIO27
+GP28,GPIO28
```

Comment:
Do you need these aliases?  They are there for Pico boards because that's the name of the pins printed on the (back) silk screen.  On your break-out board these are not visible and not the names users will use.  They will use the XRP names.